tox.ini

```diff
@@ -23,3 +23,11 @@ ignore =
     E203  # Whitespace before ':'.
 
 import-order-style = pep8
+
+[isort]
+multi_line_output = 3
+include_trailing_comma = True
+force_grid_wrap = 0
+use_parentheses = True
+ensure_newline_before_comments = True
+line_length = 88
```

Comment:
These are to get `isort` to work nicely with `black`, yeah? Nice attention to detail!